src/highdicom/sr/templates.py

```diff
@@ -1955,19 +3248,454 @@ def __init__(
+                )
+                matches.append(matches_finding)
+            if finding_site is not None:
+                matches_finding_sites = _contains_code_items(
+                    group_item,
+                    name=codes.SCT.FindingSite,
+                    value=finding_site
+                )
+                matches.append(matches_finding_sites)
+            if tracking_uid is not None:
+                matches_tracking_uid = _contains_text_items(
+                    group_item,
+                    name=codes.DCM.TrackingUniqueIdentifier,
+                    value=tracking_uid
+                )
+                matches.append(matches_tracking_uid)
+            if graphic_type is not None:
+                if isinstance(graphic_type, GraphicTypeValues):
+                    matches_graphic_type = _contains_scoord_items(
+                        group_item,
+                        name=codes.DCM.ImageRegion,
+                        graphic_type=graphic_type
+                    )
+                else:
+                    matches_graphic_type = _contains_scoord3d_items(
+                        group_item,
+                        name=codes.DCM.ImageRegion,
+                        graphic_type=graphic_type
+                    )
+                matches.append(matches_graphic_type)
+
+            seq = PlanarROIMeasurementsAndQualitativeEvaluations.from_sequence(
+                [group_item]
+            )
+            if len(matches) == 0:
+                sequences.append(seq)
+            else:
+                if all(matches):
+                    sequences.append(seq)
+
+        return sequences
+
+    def get_volumetric_roi_measurement_groups(
+        self,
+        tracking_uid: Optional[str] = None,
+        finding_type: Optional[Union[CodedConcept, Code]] = None,
+        finding_site: Optional[Union[CodedConcept, Code]] = None,
+        reference_type: Optional[Union[CodedConcept, Code]] = None,
+        graphic_type: Optional[GraphicTypeValues3D] = None,
+        referenced_sop_instance_uid: Optional[str] = None,
+        referenced_sop_class_uid: Optional[str] = None
+    ) -> List[VolumetricROIMeasurementsAndQualitativeEvaluations]:
+        """Get imaging measurement groups of volumetric regions of interest.
+
+        Finds (and optionally filters) content items contained in the
+        CONTAINER content item "Measurement group" as specified by TID 1411
+        "Volumetric ROI Measurements and Qualitative Evaluations".
+
+        Parameters
+        ----------
+        tracking_uid: str, optional
+            Unique tracking identifier
+        finding_type: Union[highdicom.sr.CodedConcept, pydicom.sr.coding.Code], optional
+            Finding
+        finding_site: Union[highdicom.sr.CodedConcept, pydicom.sr.coding.Code], optional
+            Finding site
+        reference_type: Union[highdicom.sr.GraphicTypeValues, highdicom.sr.GraphicTypeValues3D], optional
```

Comment:
I think this is a mistake
```suggestion
        reference_type: Union[highdicom.sr.CodedConcept, pydicom.sr.coding.Code], optional
```